packages/ckeditor5-font-mapper/src/FontMapper.ts

```diff
@@ -0,0 +1,117 @@
+import Plugin from "@ckeditor/ckeditor5-core/src/plugin";
+import Editor from "@ckeditor/ckeditor5-core/src/editor/editor";
+import ClipboardPipeline from "@ckeditor/ckeditor5-clipboard/src/clipboardpipeline";
+import Logger from "@coremedia/ckeditor5-logging/logging/Logger";
+import LoggerProvider from "@coremedia/ckeditor5-logging/logging/LoggerProvider";
+import DocumentFragment from "@ckeditor/ckeditor5-engine/src/view/documentfragment";
+import { UnpackFunction } from "./fontMapping/FontMapping";
+import ClipboardEventData from "@ckeditor/ckeditor5-clipboard/src/clipboardobserver";
+import EventInfo from "@ckeditor/ckeditor5-utils/src/eventinfo";
+import { ifPlugin } from "@coremedia/ckeditor5-common/Plugins";
+import FontMappingRegistry from "./fontMapping/FontMappingRegistry";
+import { replaceFontInDocumentFragment } from "./fontMapping/FontReplacer";
+
+export const CONFIG_KEY = "coremedia:fontMapper";
```

Comment:
Do we use this general config key name at other places? For use, e.g., in CKEditor config, it may be more suitable adding a namespaced name here, like `FONT_MAPPER_CONFIG_KEY` (but if, we should straighten this with other plugins).

Reply:
I decided for `COREMEDIA_FONT_MAPPER_CONFIG_KEY` as other plugins are using the style `COREMEDIA_{PLUGIN}_CONFIG_KEY` too.